src/main/java/org/jenkinsci/plugins/xunit/XUnitProcessor.java

```diff
@@ -69,7 +71,12 @@ public XUnitProcessor(TestType[] types, XUnitThreshold[] thresholds, int thresho
         this.extraConfiguration = extraConfiguration;
     }
 
-    public boolean performXUnit(boolean dryRun, AbstractBuild<?, ?> build, BuildListener listener)
+    public boolean performXunit(boolean dryRun, AbstractBuild<?, ?> build, BuildListener listener)
```

Comment:
Because @oleg-nenashev called it out. =) That said, I'm now putting backwards-compat callers in for `XUnitBuilder#perform` and `XUnitPublisher#perform`.